FirebaseSessions/Sources/FirebaseSessions.swift

```diff
@@ -86,6 +93,8 @@ protocol SessionsProvider {
 
     super.init()
 
+    self.settings.fetchAndCacheSettings(currentTime: Date())
```

Comment:
Yeah to Visu's point, we probably want this to attempt each time a session starts. We can do this by just moving this call to beginListening